### PR TITLE
Renaming Admin group to Editor in user management

### LINKED ISSUE #1476: Rename Admin group to Editor
As part of the user management we defined the set of groups that describe the set of powers the users have in our system. 

Some time ago we decided that we would like to change `admins` group to `editors` group.

child of #966

### PR DESCRIPTION
**Pull request by cbrgm**
**What this PR does / why we need it**:

This PR renames the admin group to editors for user management.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1476

**Special notes for your reviewer**:

Changed unit tests as well, everything is running smooth.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: api/pkg/controller/rbac/mapper.go [hunk 1]
```diff
@@ -143,12 +143,12 @@ func generateVerbs(groupName, resourceKind string) ([]string, error) {
 	//
 	// admins of a project
```
Comment (p0lyn0mial) [inline api/pkg/controller/rbac/mapper.go]: editors

File: api/pkg/controller/rbac/mapper.go [hunk 2]
```diff
@@ -143,12 +143,12 @@ func generateVerbs(groupName, resourceKind string) ([]string, error) {
...
 	// special case - admins are not allowed to delete a project
-	if strings.HasPrefix(groupName, adminGroupNamePrefix) && resourceKind == "Project" {
+	if strings.HasPrefix(groupName, editorGroupNamePrefix) && resourceKind == "Project" {
 		return []string{"create", "get", "update"}, nil
 	}
 
 	// admins of a resource
```
Comment (p0lyn0mial) [inline api/pkg/controller/rbac/mapper.go]: editors

**Review by p0lyn0mial — CHANGES_REQUESTED** [review]
please update the proposal - I think we can just remove `Admin` group name

https://github.com/kubermatic/kubermatic/blob/master/docs/proposals/user-management.md

**Comment by p0lyn0mial** [comment]
@cbrgm just update the proposal and this LGTM

@cbrgm if you are interested then it would be great to implement https://github.com/kubermatic/kubermatic/issues/1474

**Comment by cbrgm** [comment]
assigned #1474 to myself

**Review by p0lyn0mial — APPROVED** [review]
nice pull :)

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**